README.rst

```diff
@@ -40,14 +40,19 @@ Or with `pre-commit <https://pre-commit.com/>`__ in the ``repos`` section of you
         hooks:
         -   id: django-upgrade
             args: [--target-version, "4.1"]   # Replace with Django version
+            types: [file, python]
 
-
-To upgrade your entire project run:
+To upgrade your entire project immediately, you can tell pre-commit to run the ``django-upgrade`` hook on all files. Once you commit these changes, your other pre-commit linters will execute and you may see further changes. Use the following command:
 
 .. code-block:: sh
 
     pre-commit run django-upgrade --all-files
 
+.. tip:: Leaving this hook in place after the upgrade process is considered entirely safe and actually recommended! Django-upgrade may receive future improvements and thus keep improving your code.
```

Comment:
the tip block doesn't have any special rendering on GitHub so I'd rather just use a paragraph which starts with bold “tip”

Reply:
Agreed, didn't have time to review the GitHub rendering, will fix this.